sdk-utils/build.gradle

```diff
@@ -5,6 +5,7 @@ apply plugin: 'kotlin-kapt'
 
 android {
   defaultConfig {
+    consumerProguardFiles "proguard-rules.pro"
```

Comment:
What is the use of this ?

Reply:
As a library this is required to apply its proguard rules to consumers (sample app, MAG SDK, etc.).